modules/creating-image-expiration-notification.adoc

```diff
@@ -0,0 +1,82 @@
+// module included in the following assemblies:
+
+// * use_quay/master.adoc
+// * quay_io/master.adoc
+
+:_content-type: CONCEPT
+[id="creating-image-expiration-notification"]
+= Creating an image expiration notification
+
+Image expiration event triggers can be configured to notify users through email, Slack webhooks, and so on, and can be configured at the repository level. Triggers can be set for images expiring in any amount of days, and can work in conjunction with the auto-pruning feature.
+
+Image expiration notifications can be set by using the {productname} v2 UI or by using the `createRepoNotification` API endpoint.
+
+.Prerequisites
+
+* You have set `FEATURE_GARBAGE_COLLECTION: true` in your `config.yaml` file.
+
+.Procedure
+
+. On the {productname} v2 UI, click *Repositories*.
+
+. Select the name of a repository.
+
+. Click *Settings* -> *Events and notifications*. 
+
+. Click *Create notification*. The *Create notification* popup box appears.
+
+. Click the *Select event...* box, then click *Image expiry trigger*.
+
+. In the *When the image is due to expiry in days* box, enter the number of days before the image's expiration when you want to receive an alert. For example, use `1` for 1 day.
+
+. In the *Select method...* box, click one of the following:
++
+* E-mail
+* Webhook POST
+* Flowdock Team Notification
+* HipChat Room Notification
+* Slack Room Notification 
+
+. Depending on which method you chose, include the necessary data. For example, if you chose *Webhook POST*, include the `Webhook URL`. 
+
+. Optional. Provide a *POST JSON body template*.
+
+. Optional. Provide a *Title* for your notification.
+
+. Click *Submit*. You are returned to the *Events and notifications* page, and the notification now appears.
+
+. Optional. You can set the `NOTIFICATION_TASK_RUN_MINIMUM_INTERVAL_MINUTES` variable in your `config.yaml` file if you want to notifications for expiring images to re-run automatically. For example:
```

Comment:
how about something like - if you want to over write the existing re-run default which is 5hrs?